enclave/core/calls.c

```diff
@@ -944,13 +944,13 @@ void __oe_handle_main(
 **     context from trusted thread data. When GDB does stack walking, the parent
 **     stack of an untrusted ocall will be stack of the _OE_EXIT trusted
 **     function instead of stack of oe_enter/__morestack untrusted function.
-**     Refer to the _oe_notify_ocall_start function in host side, and the
+**     Refer to the oe_notify_ocall_start function in host side, and the
 **     OCallStartBreakpoint and update_untrusted_ocall_frame function in the
 **     python plugin.
 **
 **==============================================================================
 */
-void _oe_notify_nested_exit_start(
+void oe_notify_nested_exit_start(
```

Comment:
These debugger functions can be made static since no one else is supposed to use them.

Reply:
Here is confirmation from the POSIX standard, Section 7.1.3  - http://www.open-std.org/jtc1/sc22/wg14/www/docs/n1570.pdf
**7.1.3 Reserved identifiers of the C Standard**: 
_— All identifiers that begin with an underscore and either an uppercase letter or another
underscore are always reserved for any use.
— All identifiers that begin with an underscore are always reserved for use as identifiers
with file scope in both the ordinary and tag name spaces._
--------------
**So, in short**:  You cannot define any identifiers in global scope whose names begin with an underscore, because these may conflict with hidden (private) library definitions.

Reply:
Yes, I will try to make functions static if possible. Thanks Anand.

Reply:
I don't see any instances where I can make the functions static. The function oe_notify_nested_exit_start() is called in the assembly routines enter.S and exit.S and hence I cannot make this function static.
@anakrish - Please review. Thanks.